### PR TITLE
Feat: New parser for the "ls -lan /var/lib/sss/pubconf/krb5.include.d" command

### PR DESCRIPTION
**Pull request by rasrivas-redhat**
Signed-off-by: Rahul Srivastava <rasrivas@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This is a new parser request to list all the files present in the `/var/lib/sss/pubconf/krb5.include.d` directory.

### CODE REVIEW

File: insights/specs/default.py
Comment (bfahr) [inline insights/specs/default.py:339]: @rasrivas-redhat please add the file spec to insights/specs/insights_archive.py so that the spec collected via JSON can be accessed from the archive.  Use the `mangle` command to determine what file path to use.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/default.py
Comment (rasrivas-redhat) [reply insights/specs/default.py:339]: @bfahr - Thanks for noticing it, added related spec to insights/specs/insights_archive.py

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

**Pull request merged**
